code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -58,7 +58,7 @@ This file's folder contains:
 		return 0
 	if(M.mind.enslaved_to)
 		return 0
-	if(isdrone(M))
+	if(isdrone(M) && !istype(M, /mob/living/simple_animal/drone/cogscarab))
```

Comment:
what
goof no, they ARE drones and should return TRUE if isdrone is checking them, but I don't want them to be ineligible for being servants.